### ISSUE TITLE
flycheck has troubles with CL style cl-defun &key

### ISSUE THREAD

**Issue by marcoxa** [purcell/flycheck-package#13]
Hi

I have something like
```
(cl-defun foo (&key ((:this *that*) 42))
    "Use *THAT* (via the keyword :THIS)"
    ...)
```

flycheck things that `:this` is a variable that is not mentioned in the docstring; which is incorrect.

All the best

Marco

**Comment by purcell** [comment]
`flycheck-package` (this repo) only reports issues produced by `package-lint`, and neither would warn about that specific issue. The message you're referencing must have come from a different flycheck backend.

**Comment by marcoxa** [comment]
I stand corrected.  I guess I need to nag the flycheck-mode people.

**Comment by purcell** [comment]
Take a look at the message and see which flycheck backend is reporting the issue. `M-x flycheck-list-errors` shows a buffer in which the backend is visible. 👍
